.github/actions/rust-cache/action.yml

```diff
@@ -5,6 +5,51 @@ inputs:
     description: 'A unique identifier that will be used as part of the cache key'
     required: true
     type: string
+  # Why do we need a "restore-strategy" input field?
```

Comment:
o yeah is log, but cool , i remember it wasn't easy to explain , thanks for adding it